tests/php/SecurityAdminTest.php

```diff
@@ -50,7 +50,7 @@ class SecurityAdminTest extends FunctionalTest
 
     public function testPermissionFieldRespectsHiddenPermissions()
     {
-        $this->session()->set('loggedInAs', $this->idFromFixture(Member::class, 'admin'));
+        $this->logInAs($this->idFromFixture(Member::class, 'admin'));
```

Comment:
```suggestion
        $this->logInAs('admin');
```

Reply:
It's supposed to be using a specific fixture

Reply:
Updated